lib/taurus/qt/qtgui/graphic/taurusgraphic.py

```diff
@@ -39,6 +39,8 @@
 
 import Queue
 
+from PyQt4 import Qt
```

Comment:
Ok, I think it's solved in my last addition to this PR.

Thanks for the review!